nucliadb/setup.cfg

```diff
@@ -1,19 +0,0 @@
-[flake8]
-max_line_length = 120
-no-accept-encodings = True
-ignore =
-  E302
-  W391
-  E701
-  W504
-  F901
-  E252
-  W503
-  E203
-  BLK100
-  F541
-
-[tool:pytest]
```

Comment:
Not sure you need to remove this too

Reply:
it's moved in the toml file , see https://docs.pytest.org/en/7.3.x/reference/customize.html#pyproject-toml

Reply:
oh I forgot to commit it in the PR, forgot a `git add` :D sorry